logging/bin/deploy_fluentbit_azmonitor.sh

```diff
@@ -1,46 +1,55 @@
-#! /bin/bash
+#!/bin/bash
 
 # Copyright © 2020, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-cd "$(dirname $BASH_SOURCE)/../.."
+# shellcheck source=/dev/null
+cd "$(dirname "$BASH_SOURCE")/../.." || exit 1
```

Comment:
I looked into this quoting. Apparently bash knows the resolve the variable resolution and the quoting inside the quotes dont break anything.

Reply:
This was recommended by shellcheck, but I added it. It basically will exit the script if the "cd" command fails to execute.